# Empty
string location_name
uint16 radius
---
bool  result
uint8 aed_num

uint16 dist

string location_name
#string prefecture
string city
string address_area

float64 latitude
float64 longitude

